pkg/handlers/config.go

```diff
@@ -64,6 +64,69 @@ type CurrentAppConfigResponse struct {
 	ConfigGroups []kotsv1beta1.ConfigGroup `json:"configGroups"`
 }
 
+type DownloadAppConfigResponse struct {
```

Comment:
let's rename the response to match the function name? `DownloadFileFromConfigResponse`